packages/portal/swagger-ui-web-component/public/petstore_v3.json

```diff
@@ -78,12 +78,12 @@
           "200": {
             "description": "Successful operation",
             "content": {
-              "application/xml": {
+              "application/json": {
                 "schema": {
                   "$ref": "#/components/schemas/Pet"
                 }
               },
-              "application/json": {
+              "application/xml": {
```

Comment:
These are being switched, is that intentional? same for below

Reply:
@kaiarrowood check me here, I believe they're just being re-ordered alphabetically (`json` before `xml`) and there's no net change

Reply:
that looks like the change, just making sure

Reply:
This is an example `.json` file, I literally copied the latest example one from swagger.io so I'm guessing that's a change they made to the sample in the last few versions. No need to worry about it.